core/src/main/groovy/noe/rhel/server/httpd/HttpdBaseOS.groovy

```diff
@@ -0,0 +1,114 @@
+package noe.rhel.server.httpd
+
+import groovy.util.logging.Slf4j
+import noe.common.utils.FileStateVault
+import noe.common.utils.JBFile
+import noe.common.utils.Library
+import noe.common.utils.Version
+import noe.server.Httpd
+/**
+ * RHEL BaseOS Httpd server
+ * @author Jan Onderka <jonderka@redhat.com> *
+ */
+@Slf4j
+class HttpdBaseOS extends Httpd {
+  protected String serviceName
+  private FileStateVault fileStateVault
+
+  HttpdBaseOS(String basedir, version) {
+    super(basedir, version)
+    this.serviceName = "httpd"
+    this.basedir = "/etc/${serviceName}"
+    this.fileStateVault = new FileStateVault()
+
+    setDefault()
+    logDirs.each { JBFile.makeAccessible(new File(getServerRoot() + it)) }
```

Comment:
Is is something what really is needed?

Reply:
No, it isn't. I had it connected in my mind with different part of code.